src/codegen/mod.rs

```diff
@@ -3466,6 +3466,9 @@ impl TryToRustTy for Type {
                     inner.into_resolver().through_type_refs().resolve(ctx);
                 let inner_ty = inner.expect_type();
 
+                let is_objc_pointer =
+                    matches!(inner_ty.kind(), TypeKind::ObjCInterface(..));
```

Comment:
@emilio This is much cleaner but [wasn't added until 1.42](https://doc.rust-lang.org/std/macro.matches.html) and [the minimum supported Rust version is 1.34.](https://github.com/rust-lang/rust-bindgen#msrv). Thoughts?

Reply:
Oh, d'oh, I thought that we were using the `matches` crate already... Sure, then let's make this just a plain `match` :)